lib/Doctrine/DBAL/Logging/PsrAdapter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Doctrine\DBAL\Logging;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * Logs every query as a debug message
+ */
+final class PsrAdapter implements SQLLogger
```

Comment:
I think, the "adapter" here is essential since unlike other implementations, this class just coverts calls from one API to another. While the SQL part is implied since the DBAL doesn't log anything else than SQL.